packages/devtools_app/lib/src/screens/inspector/inspector_screen.dart

```diff
@@ -515,3 +542,31 @@ class ErrorNavigator extends StatelessWidget {
     onSelectError(newIndex);
   }
 }
+
+class PubRootDirectorySection extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    preferences.inspector.loadCustomPubRootDirectories();
+    return ValueListenableBuilder<List<IsolateRef?>>(
+      valueListenable: serviceManager.isolateManager.isolates,
+      builder: (context, value, child) {
```

Comment:
if we never use value, replace with `_`

Reply:
@kenzieschmoll it looks like we have a lint that forces a BuildContext to be named `context` should I override that lint when we don't use it?

Reply:
we can leave context, but you can change the other two params if they aren't used:
`builder: (context, _, __) {`